docs(readme): add type to description

tmcgee123 opened the pull request:
Also, updated the copyright to this year.

tymondesigns commented on README.md:
```diff
@@ -445,12 +445,12 @@ app.controller('AppCtrl', ['$scope', function ($scope) {
 <h2 id="browser-compatibility">Browser Compatibility</h2>
 
-IE8 is not supported because I am utilising `Object.keys()`
+IE8 is not supported because I am utilizing `Object.keys()`
```
I'm english :) https://en.wiktionary.org/wiki/utilise#Alternative_forms

tymondesigns commented on README.md:
````diff
@@ -445,12 +445,12 @@ app.controller('AppCtrl', ['$scope', function ($scope) {
 
 ```js
-if (! locker.supported()) {
+if (!locker.supported()) {
````
This is how I like my coding style

tmcgee123 commented:
I've reverted and squashed my changes. I apologize, JSCS does not run on markdown files and I was unaware of the styling you had set for unary operators.

tymondesigns commented:
No worries, thanks for taking the time :+1:
